webpack.base.config.js

```diff
@@ -18,7 +18,13 @@ plugins.push(
     SLOBS_BUNDLE_ID: JSON.stringify(commit),
   }),
 );
-
+plugins.push(
+  new webpack.DefinePlugin({
+    SLOBS_SENTRY_URL_FE_DSN: JSON.stringify(process.env.SLOBS_SENTRY_URL_FE_DSN),
+    SLOBS_SENTRY_URL_BE_SERVER: JSON.stringify(process.env.SLOBS_SENTRY_URL_BE_SERVER),
+    SLOBS_SENTRY_URL_BE_CLIENT: JSON.stringify(process.env.SLOBS_SENTRY_URL_BE_CLIENT),
```

Comment:
Probably better to just add this to the instance of `DefinePlugin` directly above this.

Reply:
done